packages/Php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/anonymous_function.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+function anonymousFunction()
+{
+    $anonymousFunction = function (&$bar) {};
+    $anonymousFunction(bar());
+    $staticAnonymousFunction = static function (&$bar) {};
+    $staticAnonymousFunction(bar());
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+function anonymousFunction()
+{
+    $anonymousFunction = function (&$bar) {};
+    $bar = bar();
```

Comment:
Does this rector make sure that the newly variable beeing introduced does not conflict/overlap with a already existing var?

Reply:
Yes.